pkg/liqonet/ipam/ipam.go

```diff
@@ -1042,6 +1037,22 @@ func (liqoIPAM *IPAM) MapEndpointIP(ctx context.Context, mapRequest *MapRequest)
 	return &MapResponse{Ip: mappedIP}, nil
 }
 
+func validateEndpointMappingInputs(clusterID, ip string) error {
+	const emptyClusterID = ""
+	// Parse IP
+	if netIP := net.ParseIP(ip); netIP == nil {
+		return fmt.Errorf("cannot parse endpointIP %s", ip)
```

Comment:
We have some errors defined in the `liqonet` package. Why not using them? As you have done for the `clusterID`.